package exceptionHandling;

import java.util.Scanner;

public class FinallyRunner {
	public static void main(String[] args) {
		try {

		Scanner scanner = new Scanner(System.in);

		int[] i = { 1, 3, 5, 2 };
		int num = i[5];

		scanner.close();
	} catch (ArrayIndexOutOfBoundsException e) {
		System.out.println("ArrayIndexOutOfBoundsException");
		e.printStackTrace();
	}
	}
}
